root/release_group/ReleaseGroupIndex.js

```diff
@@ -0,0 +1,196 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {withCatalystContext} from '../context';
+import Annotation from '../static/scripts/common/components/Annotation';
+import WikipediaExtract
+  from '../static/scripts/common/components/WikipediaExtract';
+import CritiqueBrainzLinks from '../components/CritiqueBrainzLinks';
+import CritiqueBrainzReview
+  from '../static/scripts/common/components/CritiqueBrainzReview';
+import PaginatedResults from '../components/PaginatedResults';
+import TaggerIcon from '../static/scripts/common/components/TaggerIcon';
+import loopParity from '../utility/loopParity';
+import EntityLink from '../static/scripts/common/components/EntityLink';
+import FormRow from '../components/FormRow';
+import FormSubmit from '../components/FormSubmit';
+import Relationships from '../components/Relationships';
+import ReleaseDates from '../components/ReleaseDates';
+import ReleaseCountries from '../components/ReleaseCountries';
+import ReleaseLabelList from '../components/ReleaseLabelList';
+import ReleaseCatnoList from '../components/ReleaseCatnoList';
+import formatBarcode from '../static/scripts/common/utility/formatBarcode';
+import * as manifest from '../static/manifest';
+import releaseGroupType from '../utility/releaseGroupType';
+
+import ReleaseGroupLayout from './ReleaseGroupLayout';
+
+type Props = {|
+  +$c: CatalystContextT,
+  +eligibleForCleanup: boolean,
+  +mostPopularReview: CritiqueBrainzReviewT,
+  +mostRecentReview: CritiqueBrainzReviewT,
+  +numberOfRevisions: number,
+  +pager: PagerT,
+  +releaseGroup: ReleaseGroupT,
+  +releases: $ReadOnlyArray<$ReadOnlyArray<ReleaseT>>,
+  +wikipediaExtract: WikipediaExtractT,
+|};
+
+function buildReleaseStatusTable($c, releaseStatusGroup) {
+  const status = releaseStatusGroup[0].status;
+  return (
+    <>
+      <tr className="subh">
+        {$c.user_exists ? <th /> : null}
+        <th colSpan={$c.session && $c.session.tport ? 9 : 8}>
+          {status && status.name
+            ? lp_attributes(status.name, 'release_status')
+            : l('(unknown)')}
+        </th>
+      </tr>
+      {releaseStatusGroup.map((release, index) => (
+        <tr className={loopParity(index)} key={release.id}>
```

Comment:
Just notice now that results converted to React have their first entry shaded whereas templates did shade the second entry. That is probably because first entry is of index 0 and thus marked as “even”. This was less noticeable with search results as these usually count more than one entry. Not sure it such an issue, anyway can be fixed globally in a later pull request if necessary.

Reply:
Yeah, I noticed this when I converted one of my first tables - it's always been like this though for React, so by now it probably doesn't matter in any way :)